pom.xml

```diff
@@ -156,7 +156,7 @@ The MIT License (MIT)
                -->
               <failOnViolation>false</failOnViolation>
               <signaturesFiles>
-                <signaturesFile>./src/test/resources/forbidden-apis.txt</signaturesFile>
+                <signaturesFile>./src/verifier/forbiddenapis.txt</signaturesFile>
```

Comment:
@neonailol why not use `./src/test/resources/forbidden-apis.txt` like in #882

Reply:
Because it is not part of sources, just like verifications.xml

Reply:
@neonailol I don't understand. `src/test/resources` is exactly the path for these kinds of things. It won't be exported to production when released.

Reply:
@llorllale well `src/test/resources` as stated from the path, is the directory for resources used by tests, this, on the other hand, is the resources needed to perform static verification of all sources. I expect in the future someone will be using this to prohibit uses of some other methods in library code, not just the tests.

Reply:
@neonailol we'll do that when the need arises. For now, please put the file back in `src/test/resources`, thanks